modules/machineset-manually-scaling.adoc

```diff
@@ -7,6 +7,11 @@
 
 If you must add or remove an instance of a machine in a MachineSet, you can manually scale the MachineSet.
 
+[NOTE]
+====
+If you have enabled the cluster-wide proxy and scale up workers after installation, you must add the workers to the Proxy object's `noProxy` field to prevent connection issues.
```

Comment:
~have~

Do you have an assembly about modifying the proxy? If so, I suggest adding a link to it and a reminder about this need to the next steps for the scale up assemblies.

Reply:
@kalexand-rh I would love to add a link, but this is in a module so I can't here.

Reply:
Sorry @kalexand-rh, I was so cranky about no xrefs in modules that I didn't fully read your suggestion about link being in the next steps of the assembly. I'll look at this.

Reply:
I understand the xref crankiness, but I think that the end of the assembly's the best place for that link anyway. :)